celery/apps/worker.py

```diff
@@ -19,7 +19,6 @@
 
 from celery import VERSION_BANNER, platforms, signals
 from celery.app import trace
-from celery.exceptions import WorkerShutdown, WorkerTerminate
```

Comment:
removing them causes no breakage right?

Reply:
The two exception types aren't used in the this module anymore, and __all__ prevents them from being cross imported. The exceptions themselves are still used at a higher level (e.g. by `maybe_shutdown()`), and there's no new additional untested code paths. Unless there's a unit test or a mystery edge case that medically needs the signal handler to throw an exception, I think it's okay.